.travis.yml

```diff
@@ -10,6 +10,6 @@ python:
 install:
     - pip install .
 script:
-    - python setup.py test -q
+    - python setup.py -q test -q
```

Comment:
We could add a matrix which enabled the `py27-subunit` and `pypy-subunit` environments.